src/sentry/interfaces/threads.py

```diff
@@ -54,7 +55,8 @@ def export_thread(data):
                 "current": data["current"],
                 "crashed": data["crashed"],
                 "name": data["name"],
-                "state": data.get("state"),
+                "state": data["state"],
+                "held_locks": data["held_locks"],
```

Comment:
Are these two always present? If not, we should use `get`.

Reply:
ah ok, I didn't know that, will change!

Reply:
I'm not familiar with this part of the codebase so I can't tell, unfortunately.